tests/pw/pages/productsPage.ts

```
@@ -391,6 +391,113 @@ export class ProductsPage extends AdminPage {
+            // min-max
+            await this.multipleElementVisible(productsVendor.minMax);
+
+            // advertisement
+            await this.toBeVisible(productsVendor.advertisement.needsPublishNotice);
+
+            // eu compliance
+            // todo: need a check for germanized plugin
+            // const { deliveryTime, optionalMiniDescription, ...euComplianceFields } = productsVendor.euComplianceFields;
+            // await this.multipleElementVisible(euComplianceFields);
+        }
+    }
```

Biome

[error] Duplicate class member name "vendorAddNewProductRenderProperly"



(lint/suspicious/noDuplicateClassMembers)